### PR TITLE
feat(local development): add vscode:clean script

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR adds a new script to the root `package.json` called `vscode:clean` which resets `lib/vscode`. 

## Changes
- add script to `./ci/dev/clean-vscode.sh`
- add script `vscode:clean` to `package.json` at root

## Checklist
- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

Fixes N/A (no accompanying issue)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "vscode": "./ci/dev/vscode.sh",
     "vscode:patch": "./ci/dev/patch-vscode.sh",
     "vscode:diff": "./ci/dev/diff-vscode.sh",
+    "vscode:clean": "./ci/dev/clean-vscode.sh",
```
Comment (nhooyr) [inline package.json:16]: Let's call this `reset`. `clean` would delete all build artifacts and caches which this doesn't do. it just resets the source.

**Comment by nhooyr** [comment]
🔥 this is going to be really helpful.

File: package.json (hunk 1 above)
Comment (jsjoeio) [reply package.json:16]: ah gotcha! good point. I'll update now.

**Review by nhooyr — APPROVED** [review]
🔥

**Comment by nhooyr** [comment]
@jsjoeio make sure to squash that second commit into the first!

Highly recommend using `git commit --fixup` to make it clear that's the intent.

**Comment by jsjoeio** [comment]
> @jsjoeio make sure to squash that second commit into the first!
> 
> Highly recommend using `git commit --fixup` to make it clear that's the intent.

😮 Thanks for the tip! I haven't used that before. I'll make those changes before I merge.

**Pull request merged**